sycamore/transforms/partition.py

```diff
@@ -23,9 +23,9 @@ def _elements_reorder_comparator(element1: Element, element2: Element) -> int:
     # following function checks if the x0 point of the element is in the
     # left column
     def element_in_left_col(e: Element) -> bool:
-        width = e.properties["coordinates"]["layout_width"]
-        x0 = e.properties["coordinates"]["points"][0][0]
-        return x0 / width <= 0.5
+        if e.bbox is None:
+            raise RuntimeError("Element BBox is None")
```

Comment:
Do we want to raise an error or fail safe?

Reply:
boundinbox/coordinates for element should not be None at least based on current algorithm, when you say fail safe, how do we behave here? return false/true by default?